app/views/profile/users/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Edit Profile</h1>
+<%= @user.name %>
+<%= @user.email %>
+<%= form_tag("/profile/edit/#{current_user.id}", method: :patch) do %>
+<%= label_tag :bio %>
+<%= text_field_tag :bio, "#{@user.bio}"%>
+<h2>DIET</h2>
+  <%= check_box_tag :vegetarian, nil, @restrictions.include?('vegetarian') %>
+  <%= label_tag "vegetarian" %>
+  <%= check_box_tag :keto, nil, @restrictions.include?('keto') %>
+  <%= label_tag "keto" %>
+  <%= check_box_tag :vegan, nil, @restrictions.include?('vegan')%>
+  <%= label_tag "vegan" %>
+  <%= check_box_tag :gluten_free, nil, @restrictions.include?('gluten free') %>
+  <%= label_tag "gluten free" %>
+  <%= check_box_tag :dairy_free, nil, @restrictions.include?('dairy free') %>
+  <%= label_tag "dairy free" %>
+  <%= submit_tag "Update Profile", :class => "btn btn-primary" %>
+<% end %>
```

Comment:
Leaving a comment here as a reminder to look into having user model methods for .vegan?/.keto? etc. instead of passing in a separate array of restrictions.